### PR TITLE
Show org dropdown for postless users

### LINKED ISSUE #9066: Occasionally org admins cannot view org dashboard
**Describe the bug**

When a user has been invited to a DEV Organization and made an Admin if they have never created a post they will not see the dropdown to switch to another Organization (screenshot 1) in their dashboard.

**Expected behavior**

All org admins should be able to access their org's dashboard regardless of whether or not they've written a post.

**Screenshots**

Screenshot 1 (_Note: I didn't doctor my "total post views at all_)

Image 2020-07-01 at 5 55 29 PM

**Additional Context**

Based on feedback [here](https://github.com/thepracticaldev/dev.to/issues/8920#issuecomment-652548734).

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Bug Fix

## Description

In `app/views/dashboards/show.html.erb` we have an `if` that removes some UI elements (post sort order and more) if the user has no posts. Alas the removed elements also include the org switcher, so org admins who don't have posts of their own won't be able to switch. I moved the `if` around as PoC, but has issues too:

1. There's now no margin between the content and the footer when the user has no posts.
2. We're displaying the sort order dropdown even if there are no articles.

I'll leave it up to @ludwiczakpawel what to do with this, i.e. add some styling here or close out this PR and open a better one, as I'm about to go offline.

## Related Tickets & Documents

Closes #9066 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/dashboards/show.html.erb [hunk 1]
```diff
@@ -13,30 +13,30 @@
...
-              <% if @articles.any? {|article| article.archived } %>
-                <%= link_to "Show archived", "javascript:;", id: "toggleArchivedLink", class: "crayons-btn crayons-btn--outlined ml-2 whitespace-nowrap my-1" %>
-              <% end %>
+      <div class="flex items-center justify-between mb-3">
+        <h2 class="hidden m:block fs-l fw-heavy">Posts</h2>
 
-              <%= select_tag "dashboard_sort", options_for_select(sort_options, params[:sort]), 'aria-label': "Sort By", class: "crayons-select w-auto ml-2 my-1" %>
+        <div class="flex flex-wrap">
+          <% if params[:which] != "organization" %>
+            <% if @articles.any?(:archived) %>
```
Comment (ludwiczakpawel) [inline app/views/dashboards/show.html.erb:21]: I can see you have changed this IF statement but now the button doesn't appear at all for me (even though I do have archived posts)

Comment (citizen428) [reply app/views/dashboards/show.html.erb:21]: Ugh, typo, it's missing a `&`.

Comment (citizen428) [reply app/views/dashboards/show.html.erb]: This has been fixed.

**Comment by michael-tharrington** [comment]
Y'all rock! 😀

**Pull request merged**